### PR TITLE
Add in-plugin education for graph block

### PR DESCRIPTION
**Pull request by truongwp**
Screenshot of the block:

Screenshot 2024-01-11 at 21 52 41

### CODE REVIEW

**Comment by truongwp** [comment]
@stephywells Please give me the addon ID of the charts addon.

**Comment by truongwp** [comment]
Thanks. I updated it. It's the download ID in the addon. The ID of Modal addon is too low, which makes me think that it's a different ID, not download ID.

**Comment by stephywells** [comment]
@truongwp 185013 is the modal add-on, so it's right. That one was been around for a long time before your block version.

File: js/src/form/graph.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		title: blockName,
+		description: __( 'Display a graph', 'formidable' ),
+		icon: FormidableIcon,
+		category: 'design',
+
+		edit: ( { setAttributes, attributes } ) => {
+			const imageWrapperStyles = {
+				padding: '38px',
+				textAlign: 'center',
+				backgroundColor: '#f2f4f7',
```
Comment (stephywells) [inline js/src/form/graph.js:38]: This doesn't match the design with a background color.  

This looks good to me:
```
				padding: '38px',
				margin: '0 auto',
				maxWidth: '600px'
```

File: js/src/form/graph.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+				<div className="frm-block-intro-screen">
+					<div className="frm-block-intro-content">
+						<FormidableIcon></FormidableIcon>
+						<div className="frm-block-title">{ blockName }</div>
+						<div className="frm-block-selector-screen frm_pro_tip" style={ { alignSelf: 'stretch' } }>
+							{ formidable_form_selector.chartsAddon.hasAccess ? (
+								<UpgradeNotice
+									text={ __( 'This site does not have Formidable Charts active.', 'formidable' ) }
+									buttonText={ __( 'Install Formidable Charts', 'formidable' ) }
+									link={ formidable_form_selector.modalAddon.link }
```
Comment (stephywells) [inline js/src/form/graph.js:52]: There are references to "modal" in here.

File: js/src/form/graph.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+	const blockName = formidable_form_selector.name + ' ' + __( 'Graph', 'formidable' );
+
+	registerBlockType( 'frm-charts/graph', {
+		title: blockName,
+		description: __( 'Display a graph', 'formidable' ),
+		icon: FormidableIcon,
+		category: 'design',
+
+		edit: ( { setAttributes, attributes } ) => {
```
Comment (stephywells) [inline js/src/form/graph.js:34]: Looks like setAttributes, attributes are unused

**Review by stephywells — CHANGES_REQUESTED** [review]

File: js/src/form/modal.js [hunk 4]
```diff
@@ -19,7 +19,11 @@ const UpgradeNotice = ( { text, buttonText, link } ) => (
 registerBlockType( 'frm-modal/modal', {
 	title: formidable_form_selector.name + ' ' + __( 'Modal', 'formidable' ),
 	description: __( 'Display a modal', 'formidable' ),
-	icon: FormidableIcon,
+	icon: (
+		<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 25 23">
```
Comment (stephywells) [inline js/src/form/modal.js:23]: The graph icon is on the modal block

**Comment by truongwp** [comment]
@stephywells I fixed those. Probably I was sleepy when creating this.

**Comment by stephywells** [comment]
Thanks for the updates @truongwp ! Can you please add screenshots in the pr description of the new block, and what it looks like in the block list now? That way, I don't think I'll need to test it again.

**Comment by truongwp** [comment]
@stephywells I added it.

**Comment by stephywells** [comment]
Sorry @truongwp , I meant reviews for QA from Garret and final review from Mike.

**Comment by truongwp** [comment]
Sorry. I'm misunderstood. I did it.

**Review by garretlaxton — APPROVED** [review]
This looks good!

File: js/src/form/graph.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+	);
+
+	const blockName = __( 'Formidable Chart', 'formidable' );
+
+	registerBlockType( 'frm-charts/graph', {
+		title: blockName,
+		description: __( 'Display a chart or graph', 'formidable' ),
+		icon: (
+			<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 25 23">
+				<path stroke="#282F36" strokeLinecap="round" strokeWidth="2.5" d="M23 2v19m-7-7.5V21M9 10.8V21m-7-2.9V21"/>
```
Comment (stephywells) [inline js/src/form/graph.js:33]: Looks like this needs to be set to stroke="currentColor" so it gets the hover color and will always match the other icons.

Comment (truongwp) [reply js/src/form/graph.js]: I updated it. I'll also update this icon in the Charts addon too.

Comment (stephywells) [reply js/src/form/graph.js]: Thank you!

**Review by Crabcyborg — APPROVED** [review]
This looks good to me. Thanks @truongwp @stephywells and @garretlaxton for working on this!

🚀

**Comment by Crabcyborg** [comment]
@stephywells It looks like I can't merge this without your approval since you requested changes.

**Review by stephywells — APPROVED** [review]
Thanks for the updates Truong!

**Pull request merged**